### DISCUSSION TITLE
Replacing Placeholder Text with Correct License Text for Issue #3738

### DISCUSSION THREAD

**Discussion by TanishMoral11** [oppia/oppia-android#5531] [Q&A (Other)] [answered]
Hi everyone,
I'm currently working on issue #3738, which involves ensuring the entire license text is displayed correctly for all copyright licenses. At the moment, there is placeholder text ('lorem ipsum') being used for testing purposes. Could anyone guide me on where I can find the official license texts for the relevant copyright licenses to replace this placeholder text? Any references or specific files in the codebase that I should look at would be very helpful.
Thanks!

**Comment by Rd4dev** [comment, marked as answer]
Hi @TanishMoral11, 

You can find the license text by following the 
- [scrapable_link](https://github.com/oppia/oppia-android/blob/8d44bf7b006aae8fbd0bab5db609ec76cd481151/scripts/assets/maven_dependencies.textproto#L1826) 
- [extracted_copy_link](https://github.com/oppia/oppia-android/blob/8d44bf7b006aae8fbd0bab5db609ec76cd481151/scripts/assets/maven_dependencies.textproto#L1836) 
- [direct_link_only](https://github.com/oppia/oppia-android/blob/8d44bf7b006aae8fbd0bab5db609ec76cd481151/scripts/assets/maven_dependencies.textproto#L1130)

in the [maven_dependencies.textproto](https://github.com/oppia/oppia-android/blob/develop/scripts/assets/maven_dependencies.textproto) file. 

#

**Apache License Text:** https://www.apache.org/licenses/LICENSE-2.0.txt
